Enterprise-CMCS/macpro-mdct-mcr
# Auth Timeout bug leak

### Description
When MCR times out, there's an error about setting a state change when the component has been unmounted. This is due to the fact that the timer is still  trying to update the timer prompt when the user is logged out.

Also fixed a similar bug but for entering an MCR form as the state was trying to set when the page has already changed.

### Related ticket(s)


MDCT-2410

---
### How to test

1) Clear cache
2) Open browser console log
3) Sign in MCR
4) Wait until timer prompt is open and you are logged out
5) Check to log to see if warning in ticket is there.

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
